cookbooks/swift/templates/default/etc/swift/object-server/default.conf-template.erb

```diff
@@ -3,6 +3,8 @@ mount_check = false
 disable_fallocate = true
 workers = 1
 log_facility = LOG_LOCAL2
+# NOTE: servers_per_port necessary for replicator/reconstructor so must go here
+servers_per_port = <%= @servers_per_port %>
```

Comment:
Well... that's not really what I meant.  I mean I just need to finish loading my brain.

Like half the time I can't even remember why the replicator needs to have a *port* much less the bind_ip and the servers_per_port integer...  I assume it has something to do with "how the fuck do i know if this thing in the ring is mine" - which IIRC is harder on a swift-all-in-one than on a real server with real ips - but I'm sure I'll be able to figure it out.

So this was really just a note to myself that I might be able to improve the comment here after I understand the *why*